packages/toolpad-core/src/DashboardLayout/DashboardSidebarSubNavigation.tsx

```diff
@@ -141,6 +141,7 @@ function DashboardSidebarSubNavigation({
 
           return (
             <Divider
+              component="li"
```

Comment:
Can we just move this to after the `key` prop?

Reply:
sure, _leave the campground cleaner than you found it_ right? ;) 

Wouldn't a linter help with proper sorting e.g. https://github.com/rkristelijn/eslint-plugin-mui

Reply:
It would help, I guess we don't want to strictly enforce that but at least I like the prop order to be as consistent as possible :)